ScienceSaturday04/pizza_results.py

```diff
@@ -0,0 +1,135 @@
+import pandas as pd
+import numpy as np
+import matplotlib
+matplotlib.use('agg')
+import matplotlib.pyplot as plt
+
+matplotlib.style.use('bmh')
+
+def read_tables():
+
+    #Reading in cheese results and creating pandas dataframe
+    f1  = pd.read_excel('SS4_pizza_tables.xlsx',
+                        sheet_name='Cheese Results',
+                        header=0,
+                        usecols='C:K'
+                        )
+
+    pizzas = ['Pappa John\'s','Pizza Boli\'s','Digiorno','Little Caesar\'s','Dominos','Pizza Hut','Pizza Hut Stuffed Crust']
```

Comment:
No need for escape characters for apostrophes if you use double quotes (`" ' "`)

Reply:
Changed those.